vaadin-combo-box-flow-integration-tests/src/test/java/com/vaadin/flow/component/combobox/test/LazyLoadingIT.java

```diff
@@ -105,9 +108,19 @@ public void scrollToEnd_scrollUpwards_pagesLoaded() {
         scrollToItem(stringBox, 920);
 
         assertLoadedItemsCount(
-                "Expected the first and the two last pages to be loaded (150 items).",
-                150, stringBox);
+                "Expected the two last pages to be loaded (100 items).",
+                100, stringBox);
         assertRendered("Item 920");
+        
+        scrollToItem(stringBox, 870);
```

Comment:
Extended the test to also scroll backwards to make sure the requested range increases respectively